eng/perf-job.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  readyToRun: true
+  helixQueues: ''
+  container: ''
+  crossrootfsDir: ''
+  framework: netcoreapp5.0 # Specify the appropriate framework when running release branches (ie netcoreapp3.0 for release/3.0)
+
+### Test job
+
+### Each test job depends on a corresponding build job with the same
+### buildConfig and archType.
+
+jobs:
+- template: /eng/common/templates/job/performance.yml
+  parameters:
+    # Compute job name from template parameters
+    jobName: ${{ format('perfbuild_{0}_{1}_{2}', parameters.osIdentifier, parameters.archType, parameters.buildConfig) }}
```

Comment:
Please also create a `displayName` that doesn't have underscores - e.g. Perf Windows_NT x64 Release.